packages/sui-mono/src/config.js

```diff
@@ -35,6 +36,15 @@ module.exports = {
   },
   getPublishAccess: function () {
     return publishAccess
+  },
+  getProjectName: function () {
+    return projectPackage.name
+  },
+  isMonoPackage: function () {
+    const folders = cwds(path.join(basePath, packagesFolder), deepLevel)
+    const scopePath = folders.pop()
+
```

Comment:
@carlosvillu As we agreed I'll check all the scopes and if one does not have the package.json is not a multi package.

I just didn't have the time to do it

@davidbarna I'm not sure if I understood this.

If deeplevel = 0 we wouldn't have any scope. and we may still want scopes in a mono package repo.

We use 1 scope 1 package but in a mono package scopes are informative..

"I've made changes in the Url parsing module" kind of